.github/workflows/known-FPs.csv

```diff
@@ -44,3 +44,4 @@ fdbf0b9d-0182-4c43-893b-a1eaab92d085;Newly Registered Protocol Handler;.*
 52a85084-6989-40c3-8f32-091e12e17692;Suspicious Usage of CVE_2021_34484 or CVE 2022_21919;Computer: Agamemnon
 573df571-a223-43bc-846e-3f98da481eca;Copy a File Downloaded From Internet;7z\.exe
 37774c23-25a1-4adb-bb6d-8bb9fd59c0f8;Image Load of VSS Dll by Uncommon Executable;SetupFrontEnd\.exe
+801bd44f-ceed-4eb6-887c-11544633c0aa;Windows Defender Suspicious Configuration Changes;.*
```

Comment:
I guess this happens a lot during updates. Is that the case? If yes the rule cannot be level high.

Reply:
I removed the filter for you to check that it's only one value which is the tamper protection. I don't know how the logs were set up. But the idea of the rule is to check the changes to sensitive values (even if it's just an update)

Reply:
I remove the `Tamper Protection` value until further tests. To confirm when it happens.